diff --git a/dist/StorybookResolver.js b/dist/StorybookResolver.js
index 3e97e9267df366cfbceeae3a03ac3058d1c1cafd..94f8816894f08ed578727a90d028cef663336acf 100644
--- a/dist/StorybookResolver.js
+++ b/dist/StorybookResolver.js
@@ -17,7 +17,14 @@ module.exports = new _plugin.Resolver({
     async resolve ({ dependency, options, specifier, pipeline, logger }) {
         // Workaround for interop issue
         if (specifier === "react-dom/client") {
-            let specifier = REACT_MAJOR_VERSION >= 18 ? "react-dom/client.js" : "react-dom/index.js";
+            let specifier;
+            if (REACT_MAJOR_VERSION >= 19) {
+                specifier = "react-dom/client";
+            } else if (REACT_MAJOR_VERSION >= 18) {
+                specifier = "react-dom/client.js";
+            } else {
+                specifier = "react-dom/index.js";
+            }
             return {
                 filePath: __dirname + "/react.js",
                 code: `
